vsphere/internal/virtualdevice/virtual_machine_tag_rules_subresource.go

```diff
@@ -0,0 +1,27 @@
+package virtualdevice
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func VirtualMachineTagRulesSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"tag_category": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "category",
+		},
+		"tags": {
+			Type:        schema.TypeList,
+			Required:    true,
+			Description: "tags",
```

Comment:
description is cryptic..can you make it more descriptive? Applies to line 12 also..